repository-data/application/src/main/resources/hcm-config/hst/configurations/publicationsystem/components/publications-overview-component.yaml

```diff
@@ -0,0 +1,5 @@
+definitions:
```

Comment:
we don't need this file. There is no additional configuration. Just use `hst:componentclassname: uk.nhs.digital.ps.components.PublicationsOverviewComponent` in page config

Reply:
Ah, cool, indeed. Configuration now moved to `hcm-config/hst/configurations/publicationsystem/pages/home.yaml` with the `publications-overview-component.yaml` removed.